config.c

```diff
@@ -125,6 +125,20 @@ void read_FTLconf(void)
 	else
 		logg("   RESOLVE_IPV6: Don\'t resolve IPv6 addresses");
 
+	// RESOLVE_IPV4
+	// defaults to: Yes
+	config.resolveIPv4 = true;
+	buffer = parse_FTLconf(fp, "RESOLVE_IPV4");
+	if(buffer != NULL)
+	{
+		if(strcmp(buffer, "no") == 0)
+			config.resolveIPv4 = false;
```

Comment:
This could be shortened to one if statement

Reply:
Fixed by 4f7a36d